### PR TITLE
Add payee compo added

### PR DESCRIPTION
**Pull request by Nayana-2607**

### CODE REVIEW

**Comment by Rajkumar-MP** [comment]
@Nayana-2607 resolve the conflict

File: package.json [hunk 1]
```diff
@@ -36,10 +36,14 @@
...
-    "@lion/input": "^0.11.1",
-    "@lion/localize": "^0.16.0",
+    "@lion/form-core": "^0.8.4",
+    "@lion/input": "^0.11.4",
+    "@lion/localize": "^0.16.1",
+    "deep-equal": "^2.0.5",
     "lit-element": "^2.0.1",
-    "lit-html": "^1.0.0"
+    "lit-html": "^1.0.0",
+    "node-modules": "^1.0.1",
```
Comment (Rajkumar-MP) [inline package.json]: why node-mdules and webpack-dev-server added here ?

File: src/Components/fe-new-payee/FeNewPayee.js [hunk 2]
```diff
@@ -0,0 +1,128 @@
+import { LitElement, html, css } from 'lit-element';
+import { localize, LocalizeMixin } from '@lion/localize';
+import { Required } from '@lion/form-core';
+
+import '@lion/button/';
```
Comment (Rajkumar-MP) [inline src/Components/fe-new-payee/FeNewPayee.js]: is this valid ?

**Review by Rajkumar-MP — CHANGES_REQUESTED** [review]

File: src/Components/fe-new-payee/FeNewPayee.js (hunk 2 above)
Comment (Nayana-2607) [reply src/Components/fe-new-payee/FeNewPayee.js]: I will correct the code

File: package.json (hunk 1 above)
Comment (Nayana-2607) [reply package.json]: Removed the extra dependencies

File: package.json [hunk 3]
```diff
@@ -36,9 +36,11 @@
   "dependencies": {
     "@lion/button": "^0.8.11",
     "@lion/form": "^0.7.18",
+    "@lion/form-core": "^0.8.4",
     "@lion/input": "^0.11.1",
     "@lion/localize": "^0.16.0",
     "@vaadin/router": "^1.7.4",
+    "deep-equal": "^2.0.5",
```
Comment (Rajkumar-MP) [inline package.json]: why we need this ?

**Review by Rajkumar-MP — DISMISSED** [review]

File: package.json (hunk 3 above)
Comment (Nayana-2607) [reply package.json]: I will check and remove

**Review by Rajkumar-MP — DISMISSED** [review]

**Review by alekhya66 — DISMISSED** [review]

**Review by shadushazzz — DISMISSED** [review]

**Review by shadushazzz — APPROVED** [review]

**Review by alekhya66 — APPROVED** [review]

**Review by Rajkumar-MP — APPROVED** [review]

**Pull request merged**